redux_userlist_06/src/components/Dashboard/getList.tsx

```diff
@@ -0,0 +1,60 @@
+import "./styles.css";
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { addUserList } from "../../actions/actions";
+import { UserDetails } from "../../interfaces";
+import UserProfile from "./userProfile";
+import axios from "axios";
+import * as constants from "../../constants/constants";
+import { Link } from "react-router-dom";
+
+function GetList() {
```

Comment:
Please use ES6's arrow functions, they are easier to read and concise as well.
Replace All the functions with ES^ arrow functions.